spectrum/xpub.py

```diff
@@ -148,4 +187,16 @@ def choose_some(self, quantity):
         for button in range(0, quantity):
             buttons[button].click()
         self._add.click()
-        WebDriverWait(self._driver, self.TIMEOUT_CLOSING).until_not(lambda driver: self._find_picker().is_displayed())
+        WebDriverWait(self._driver, self.TIMEOUT_CLOSING).until(lambda driver: self._picker_closed())
+
+    def _picker_closed(self):
+        """When the PeoplePicker is open, there are two h2 elements on the page.
+
+        - Who should review your work?
+        - Suggest [senior|reviewing] editors
+
+        The first is invisible. We check the second has disappeared to declare the PeoplePicker is now closed."""
+        LOGGER.debug("PeoplePicker visibility check")
+        headings = self._driver.find_elements_by_css_selector(self.CSS_HEADING)
+        LOGGER.debug("Headings: %d", len(headings))
```

Comment:
Convoluted to say the least: any other positive check (element present or visible, rather than element not present) to understand whether the PeoplePicker is now closed? Could even be at the page level (`XpubInitialSubmissionEditorsPage` class) if it's unrelated to the PeoplePicker itself